### DISCUSSION TITLE
Is disabling arp scan supported?

### DISCUSSION THREAD

**Discussion by meeowgler** [netalertx/NetAlertX#699] [Q&A]
If arp scan is disabled (to use an nmap only scan, for instance), when you select a device network node, the name that is populated in the "Node (MAC)" field is "(name not found)". When re-enabling arp scan, the "Node (MAC)" field shows correctly.

Should we always keep arp scan enabled?

Thanks!

**Comment by jokob-sk** [comment]
Hi @meeowgler ,

arp-scan shouldn't be required. Can you post some screenshots to illustrate the issue?

**Reply by meeowgler** [reply]
![NetAlertX - Network name not found](https://github.com/jokob-sk/NetAlertX/assets/4227406/06f48ac5-0ec3-42d6-bc5c-f134420ba217)

**Reply by jokob-sk** [reply]
I set ARPSCAN_RUN to disabled and I can still see proper names:

![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/e8e0f41c-407c-4157-b77a-ca59b2123b47)
![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/31afaf1a-a6be-45c2-b2c3-724baa7fe3db)


Can you try refreshing the cache? 
![image](https://github.com/jokob-sk/NetAlertX/assets/96159884/19abf2eb-2477-40eb-a4c4-5b9cb39ac238)

**Comment by jokob-sk** [comment]
Hi @meeowgler - in the end it was a bug - I couldn't reproduce it the first time because I re-enabled ARPSCAN right away. You can check this issue for details: #726
